extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -374,31 +386,35 @@ public function testGetContext(
 			$expected['slots'][$var] = $val;
 		}
 
-		if ($expected['targeting']['pageType'] === 'article') {
+		if ( $expected['targeting']['pageType'] === 'article' ) {
 			$expected['providers']['taboola'] = true;
 		}
 
 		// Check for SourcePoint URL
 		$this->assertStringMatchesFormat( $expectedAdEngineResourceURLFormat, $result['opts']['sourcePointDetectionUrl'] );
-		unset( $result['opts']['sourcePointDetectionUrl'] );
+		unset($result['opts']['sourcePointDetectionUrl']);
 
 		// Check for PageFair URL
 		$this->assertStringMatchesFormat( $expectedAdEngineResourceURLFormat, $result['opts']['pageFairDetectionUrl'] );
-		unset( $result['opts']['pageFairDetectionUrl'] );
+		unset($result['opts']['pageFairDetectionUrl']);
+
+		// Check for PageFair URL
+		$this->assertEquals( $expectedPrebidBidderUrl, $result['opts']['prebidBidderUrl'] );
+		unset($result['opts']['prebidBidderUrl']);
 
 
 		$expected['providers']['rubiconFastlane'] = true;
 
 		// Extra check for Monetization Service
-		if ( isset( $expectedProviders['monetizationService'] ) ) {
+		if ( isset($expectedProviders['monetizationService']) ) {
 			$this->assertTrue( is_array( $result['providers']['monetizationServiceAds'] ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "monetizationServiceAds" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)